package.json

```diff
@@ -154,6 +154,7 @@
     "vue-infinite-loading": "2.4.5",
     "vue-router": "3.5.3",
     "vue-slider-component": "3.2.11",
+    "vue-virtual-scroller": "1.1.2",
```

Comment:
Should probably remove the no longer used one too right?

Reply:
Correct.